src/python/espressomd/lbboundaries.py

```diff
@@ -1,25 +1,27 @@
 from __future__ import print_function, absolute_import
 from .script_interface import ScriptInterfaceHelper
+import espressomd.code_info
 
 
-class LBBoundaries(ScriptInterfaceHelper):
-    _so_name = "LBBoundaries::LBBoundaries"
+if any(i in espressomd.code_info.features() for i in ["LB_BOUNDARIES", "LB_BOUNDARIES_GPU"]):
```

Comment:
How about throwing an `ImportError` instead of `if`-ing the whole class definition (which apparently fails the checks).
````python
if not set(["LB_BOUNDARIES", "LB_BOUNDARIES_GPU"]) < set(espressomd.code_info.features()):
    throw ImportError("LB_BOUNDARIES not compiled in!")

class LBBoundaries(ScriptInterfaceHelper):
    ...
````